src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-weight: 400;
+  width: 200px;
+  height: 408px;
```

Comment:
do not hardcode height - it should depend on content (and margins between content elements)